File: app/src/main/java/social/entourage/android/map/LoaderCardViewHolder.java
package social.entourage.android.map;

import android.view.View;
import android.widget.ProgressBar;
import social.entourage.android.R;
import social.entourage.android.api.model.TimestampedObject;
import social.entourage.android.base.BaseCardViewHolder;

public class LoaderCardViewHolder extends BaseCardViewHolder {
    private  ProgressBar loader;

    public LoaderCardViewHolder(View view) {
        super(view);
    }

    @Override
    protected void bindFields() {
        loader = itemView.findViewById(R.id.layout_loader_card_progressBar);
    }

    @Override
    public void populate(TimestampedObject data) {
        loader.setVisibility(View.VISIBLE);
    }
}

File: app/src/main/java/social/entourage/android/base/LoaderCardViewHolder.kt
package social.entourage.android.base

import android.view.View
import social.entourage.android.api.model.TimestampedObject
import social.entourage.android.base.BaseCardViewHolder
import kotlinx.android.synthetic.main.layout_loader_card.view.*

class LoaderCardViewHolder(view: View?) : BaseCardViewHolder(view) {
    override fun bindFields() {
    }

    override fun populate(data: TimestampedObject) {
        itemView.layout_loader_card_progressbar.visibility = View.VISIBLE
    }
}